new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -175,6 +176,11 @@ class LogFileManager {
             } due to a limitation in Chromium-based browsers.`);
         }
 
+        const result = decoder.buildIdx(0, FULL_RANGE_END_IDX)
+        if (result != null && result.numInvalidEvents > 0) {
```

Comment:
Maybe the correct logic is to check against `null` equality and use `||`
```suggestion
        if (null === result || 0 < result.numInvalidEvents) {
```

Reply:
The null check was necessary or accessing `result`(null) would cause an error